src/app/public/modules/ag-grid/ag-grid.module.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  AgGridModule
+} from 'ag-grid-angular';
+
+import {
+  SkyAgGridService
+} from './ag-grid.service';
+
+import {
+  SkyAgGridCellEditorDatepickerComponent,
+  SkyAgGridCellEditorDatepickerModule,
+  SkyAgGridCellEditorNumberComponent,
+  SkyAgGridCellEditorNumberModule
+} from './cell-editors';
+
+import {
+  SkyAgGridCellRendererRowSelectorComponent,
+  SkyAgGridCellRendererRowSelectorModule
+} from './cell-renderers';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    AgGridModule,
```

Comment:
Importing the module is probably fine, especially if you're using some of the `AgGridModule` components/services.